sdk/openai/azure-ai-openai-assistants/src/main/java/com/azure/ai/openai/assistants/implementation/AssistantsClientImpl.java

```diff
@@ -916,7 +916,8 @@ public Mono<Response<BinaryData>> createAssistantWithResponseAsync(BinaryData as
     public Response<BinaryData> createAssistantWithResponse(BinaryData assistantCreationOptions,
         RequestOptions requestOptions) {
         final String accept = "application/json";
-        return service.createAssistantSync(this.getEndpoint(), this.serviceVersion.getVersion(), accept, assistantCreationOptions, requestOptions,
+        return service.createAssistantSync(this.getEndpoint(), this.serviceVersion.getVersion(),
+                accept, assistantCreationOptions, requestOptions,
```

Comment:
This method is not annotated with `@Generated` should we restore it? Or was it ever there?

Reply:
`xxxClientImpl.java` always doesn't generated with @Generated 
https://github.com/Azure/azure-sdk-for-java/blob/main/sdk/openai/azure-ai-openai/src/main/java/com/azure/ai/openai/implementation/OpenAIClientImpl.java#L382